sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIOWriteTest.java

```diff
@@ -3025,4 +3025,50 @@ public void testWriteWithStorageApiWithoutSettingShardsEnableAutoSharding() thro
         fakeDatasetService.getAllRows("project-id", "dataset-id", "table-id"),
         containsInAnyOrder(new TableRow().set("name", "a"), new TableRow().set("name", "b")));
   }
+
+  @Test
+  public void testBatchStorageWriteWithMultipleAppendsPerStream() throws Exception {
+    assumeTrue(useStorageApi);
+    assumeTrue(!useStreaming);
```

Comment:
as in that pr, to make this test run one needs to add a parametrized combination satisfies these constraints

Reply:
never mind

I see the test is executed: https://ci-beam.apache.org/job/beam_PreCommit_Java_GCP_IO_Direct_Phrase/181/testReport/org.apache.beam.sdk.io.gcp.bigquery/BigQueryIOWriteTest/testBatchStorageWriteWithMultipleAppendsPerStream_2_/

Reply:
Current succeeded run:
https://ci-beam.apache.org/job/beam_PreCommit_Java_GCP_IO_Direct_Commit/2745/
https://ci-beam.apache.org/job/beam_PreCommit_Java_GCP_IO_Direct_Phrase/181/